apiserver/cmd/apiserver/server/server.go

```diff
@@ -53,7 +53,7 @@ func logrusLevel() logrus.Level {
 	return logrus.ErrorLevel
 }
 
-// NewCommandStartMaster provides a CLI handler for 'start master' command
+// NewCommandStartCalicoServer provides a CLI handler for 'start master' command
```

Comment:
Better just to delete this comment IMO.  The `'start master'` part is wrong, and it's really not adding any value.

Reply:
Thanks! Modified